pom.xml

```
@@ -102,10 +102,12 @@
     <fasterxml.jackson.dataformat.yaml.version>${fasterxml.spark3.version}</fasterxml.jackson.dataformat.yaml.version>
     <kafka.version>2.0.0</kafka.version>
     <pulsar.version>3.0.2</pulsar.version>
+    <kafka.spark3.version>2.8.2</kafka.spark3.version>
     <pulsar.spark.version>${pulsar.spark.scala12.version}</pulsar.spark.version>
     <pulsar.spark.scala12.version>3.1.1.4</pulsar.spark.scala12.version>
     <pulsar.spark.scala13.version>3.4.1.1</pulsar.spark.scala13.version>
     <confluent.version>5.5.0</confluent.version>
+    <connect.api.version>2.5.0</connect.api.version>
```

Move this above to the `kafka.` configs.
```suggestion
    <kafka.connect.api.version>2.5.0</kafka.connect.api.version>
```

---

done